conf/salt/project/worker/default.sls

```diff
@@ -38,4 +38,12 @@ include:
     - require:
       - file: {{ instance }}_conf
 
+nightly-restart-{{ instance }}-celery:
```

Comment:
We're using underscores rather than hyphens for the other state names in this file, not that salt cares (I don't think).